sidechain/consensus/aura/src/verifier.rs

```diff
@@ -127,14 +127,6 @@ where
 	ParentchainHeader: ParentchainHeaderTrait<Hash = BlockHash>,
 	Context: ValidateerFetch + EnclaveOnChainOCallApi,
 {
-	ensure!(
```

Comment:
the pc header peeking returns an error in case the pc header does not match the sidechain header. If that's the case, we simply return here, and the ensure! in verifier.verify can be dropped completely without leaving an opening.